packages/cyberstorm/src/components/Layout/PackageListLayout/PackageListLayout.tsx

```diff
@@ -69,17 +74,24 @@ export function PackageListLayout(props: PackageListLayoutProps) {
                 formatInteger(communityData.community.total_download_count) +
                 " Downloads"
               }
-              icon={<FontAwesomeIcon icon={faDownload} fixedWidth />}
+              icon={
+                <Icon>
+                  <FontAwesomeIcon icon={faDownload} />
+                </Icon>
+              }
               colorScheme="accent"
               size="bold_large"
             />,
-            <Link
```

Comment:
I would prefer to have some sort of explicit decision if we are to stop using the "anonymous" Link component for external links. Not just have it done "silently" as a part of huge refactoring PR.